src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Figure implements Areacounter,Drawable {

```

Comment:
```suggestion
public class Figure implements Areacounter, Drawable {
```
let's make this class abstract